webapp-admin/src/components/SystemUserForm.js

```diff
@@ -188,6 +202,20 @@ class SystemUserForm extends Component {
                             })}
                             </select>
                         </label>
+                        <label>Limit this system-user to a set of serial numbers:&nbsp;
+                            <button className="p-button--brand" title="Add serial number" onClick={this.handleAddEmptySerialField}>
+                                <i className="fa fa-plus"></i>
```

Comment:
If you are using Vanilla here we have a plus icon: `p-icon--plus`. https://vanillaframework.io/docs/patterns/icons